src/scvi/model/_scvi.py

```diff
@@ -232,6 +229,112 @@ def setup_anndata(
         adata_manager.register_fields(adata, **kwargs)
         cls.register_manager(adata_manager)
 
+    @staticmethod
+    def _get_summary_stats_from_registry(registry: dict) -> attrdict:
+        summary_stats = {}
+        for field_registry in registry[_constants._FIELD_REGISTRIES_KEY].values():
+            field_summary_stats = field_registry[_constants._SUMMARY_STATS_KEY]
+            summary_stats.update(field_summary_stats)
+        return attrdict(summary_stats)
+
+    @classmethod
+    @setup_anndata_dsp.dedent
+    def setup_datamodule(
+        cls,
+        datamodule,  # TODO: what to put here?
```

Comment:
It should be pytorch.DataLoader right? Martin has done typing for it in the current code.